langserve/callbacks.py

```diff
@@ -1,9 +1,15 @@
 from __future__ import annotations
 
+import uuid
 from typing import Any, Dict, List, Optional, Sequence
 from uuid import UUID
 
 from langchain.callbacks.base import AsyncCallbackHandler
+from langchain.callbacks.manager import (
+    BaseRunManager,
+    _ahandle_event,
```

Comment:
New release of langchain makes these public

Reply:
should we just land this after that?